[2024-08-25 17:42:39,074] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-19T00:00:00+00:00 [queued]>
[2024-08-25 17:42:39,078] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-19T00:00:00+00:00 [queued]>
[2024-08-25 17:42:39,078] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-08-25 17:42:39,078] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-08-25 17:42:39,078] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-08-25 17:42:39,090] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extrai_dados> on 2022-12-19 00:00:00+00:00
[2024-08-25 17:42:39,093] {standard_task_runner.py:52} INFO - Started process 162242 to run task
[2024-08-25 17:42:39,095] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos', 'extrai_dados', 'scheduled__2022-12-19T00:00:00+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmperqu4b5a', '--error-file', '/tmp/tmpopdnzkzr']
[2024-08-25 17:42:39,095] {standard_task_runner.py:80} INFO - Job 57: Subtask extrai_dados
[2024-08-25 17:42:39,128] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos.extrai_dados scheduled__2022-12-19T00:00:00+00:00 [running]> on host adrilucy.
[2024-08-25 17:42:39,167] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos
AIRFLOW_CTX_TASK_ID=extrai_dados
AIRFLOW_CTX_EXECUTION_DATE=2022-12-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-19T00:00:00+00:00
[2024-08-25 17:42:39,949] {python.py:173} INFO - Done. Returned value was: None
[2024-08-25 17:42:39,954] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos, task_id=extrai_dados, execution_date=20221219T000000, start_date=20240825T204239, end_date=20240825T204239
[2024-08-25 17:42:39,994] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-08-25 17:42:40,014] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
